Library/Formula/eg.rb

```diff
@@ -0,0 +1,40 @@
+class Eg < Formula
+  homepage "https://github.com/srsudar/eg"
+  url "https://github.com/srsudar/eg/archive/v0.0.3.tar.gz"
+  sha256 "c257e334f2a2e494f9b9791b92c1d3f7b00da3e2e7ec47f02d720342b466f968"
+
+  def install
+    if Dir.exist? "#{var}/examples"
```

Comment:
Are you saying the examples of other formulae should be removed if `eg` is uninstalled? I disagree, I think that's data of the other formulae that `eg` shouldn't be touching, whether it can only be read by it or not. They're data, not configuration or code.